packages/govuk-frontend/src/govuk/all.mjs

```diff
@@ -24,63 +24,63 @@ function initAll (config) {
 
   // Allow the user to initialise GOV.UK Frontend in only certain sections of the page
   // Defaults to the entire document if nothing is set.
-  var $scope = config.scope instanceof HTMLElement ? config.scope : document
+  const $scope = config.scope instanceof HTMLElement ? config.scope : document
 
-  var $accordions = $scope.querySelectorAll('[data-module="govuk-accordion"]')
-  $accordions.forEach(function ($accordion) {
+  const $accordions = $scope.querySelectorAll('[data-module="govuk-accordion"]')
+  $accordions.forEach(($accordion) => {
```

Comment:
For all these arrow function changes I'm aware they're split into:

1. Arrow functions to preserve `this` as the component instance
2. Arrow functions to avoid `.bind(this)` in various event handlers
3. Arrow functions simply for "code style" or consistency

I'm aware that there are plenty of event listeners and `.forEach()` loops (like this one) that never actually refer to `this` so may not necessary to change those outside of component scope

Terser is going to convert all that it can to arrow functions anyway to save KBs